src/fluree/db/nameservice/core.cljc

```diff
@@ -0,0 +1,91 @@
+(ns fluree.db.nameservice.core
+  (:refer-clojure :exclude [exists?])
+  (:require [clojure.string :as str]
+            [fluree.db.conn.proto :as conn-proto]
+            [fluree.db.nameservice.proto :as ns-proto]
+            [fluree.db.util.async :refer [<? go-try]]
+            [clojure.core.async :as async :refer [go]]
+            [fluree.db.util.log :as log]))
+
+#?(:clj (set! *warn-on-reflection* true))
+
+(defn nameservices
+  [conn]
+  (conn-proto/-nameservices conn))
+
+(defn relative?
```

Comment:
Very minor. I think the name of this function could be more descriptive. Upon my first scanning, I thought it was talking about if a nameservice was relative, not a ledger address. I think something like `relative-ledger-alias?` would be more clear.

Reply:
Updated in last commit.